terraform-aws-github-runner/modules/runners/lambdas/runners/src/scale-runners/runners.ts

```diff
@@ -691,7 +738,15 @@ export async function createRunner(runnerParameters: RunnerInputParameters, metr
     const tags = [
       { Key: 'Application', Value: 'github-action-runner' },
       { Key: 'RunnerType', Value: runnerParameters.runnerType.runnerTypeName },
+      { Key: 'RepositoryOwner', Value: runnerParameters.repositoryOwner },
+      { Key: 'RepositoryName', Value: runnerParameters.repositoryName },
+      { Key: 'InstanceType', Value: runnerParameters.runnerType.instance_type },
```

Comment:
Humm, 

I wonder if we need this.

When we list things on AWS, doesn't it return the instanceType of each instance we're listing? Because if it does, this information is more trustworthy and we can simplify our code a bit.